src/Data/List/Relation/Unary/Enumerates/Setoid/Properties.agda

```diff
@@ -40,8 +40,8 @@ module _ (S : Setoid a ℓ₁) (T : Setoid b ℓ₂) (surj : Surjection S T) whe
   open Surjection surj
 
   map⁺ : ∀ {xs} → IsEnumeration S xs → IsEnumeration T (map to xs)
-  map⁺ _∈xs y with strictlySurjective y
-  ... | (x , fx≈y) = ∈-resp-≈ T fx≈y (∈-map⁺ S T cong (x ∈xs))
+  map⁺ _∈xs y = case strictlySurjective y of
+                  λ where (x , fx≈y) → ∈-resp-≈ T fx≈y (∈-map⁺ S T cong (x ∈xs))
```

Comment:
I'd be tempted to use irrefutable `with` here, or even `let`, because each choices intorduces less boilerplate than `case_of_` and a pattern-matching lambda?